test/mongodb.test.js

```diff
@@ -1438,6 +1438,184 @@ describe('mongodb connector', function() {
       });
     });
 
+    it('find should be able to query by location via near with maxDistance', function(done) {
+      var coords = { lat: 30.274085, lng: 120.15507000000002 };
+
+      geoDb.autoupdate(function(err) {
+        var createPosts = function(far) {
```

Comment:
Please move this function outside of the test and share it with the other 3 tests.

```
// geopoint being {lat: i, lng: j}
function createPosts(isFar, geopoint) {
  ...
}
Then just pass in the differences.